server/api/controllers/users/controller.ts

```diff
@@ -47,6 +47,19 @@ export class Controller {
       return next(err);
     }
   }
+
+  async login(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { email, password } = req.body;
+      // validation would be handled in the User model
+      const user = await UsersService.getBy("email", email);
+      // prevent user from being null
+      const isAuth = user ? user.password == password : false;
```

Comment:
It might be useful to have some more sophisticated error handling in the future. For example, for invalid user emails we could send a response of "Invalid email".

Reply:
For security reasons, we would actually allow hackers to easily brute force passwords/email if we tell them what field has the wrong value. This is a common security measure adopted by many apps